From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Christophe Fergeau <cfergeau@redhat.com>
Date: Thu, 17 Mar 2016 12:39:38 +0100
Subject: [PATCH] storage-domain: Remove debug log on missing node

This is redundant with a similar log in
ovirt_rest_xml_node_get_content_va
---
 govirt/ovirt-storage-domain.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/govirt/ovirt-storage-domain.c b/govirt/ovirt-storage-domain.c
index a037043..4e002de 100644
--- a/govirt/ovirt-storage-domain.c
+++ b/govirt/ovirt-storage-domain.c
@@ -297,7 +297,6 @@ ovirt_resource_parse_xml(OvirtResource *resource,
 
         value_str = ovirt_rest_xml_node_get_content_from_path(node, elements->xml_node);
         if (value_str == NULL) {
-            g_debug("Could not find node %s", elements->xml_node);
             continue;
         }
         g_value_init(&value, elements->type);
